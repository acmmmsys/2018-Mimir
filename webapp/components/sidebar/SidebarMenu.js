import React, {Component} from 'react';
import classnames from 'classnames';
import 'style/sidebar/SideBar.scss';

class SidebarMenu extends Component {
    render() {
        return (
            <ul className='ui-sidebar-menu'>
                { this.props.children }
            </ul>
        );
    }
}

export default SidebarMenu;