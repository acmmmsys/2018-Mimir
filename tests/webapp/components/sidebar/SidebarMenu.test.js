import React from 'react';
import SidebarMenu from '../../../../webapp/components/sidebar/SidebarMenu'
import renderer from 'react-test-renderer';

let wrapper;

test('SidebarMenu does render', () => {
    wrapper = shallow(
        <SidebarMenu>

        </SidebarMenu>
    );
    expect(wrapper).toMatchSnapshot();
});
