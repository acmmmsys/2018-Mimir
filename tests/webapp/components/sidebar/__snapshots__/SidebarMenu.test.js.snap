// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SidebarMenu does render 1`] = `<ul />`;
